<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>259</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>四轴上位机v1.2.2.t(luyuexin)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogDataSource">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>数据源</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogScene">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>场景</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogCalibration">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>校正</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogControl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>控制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogStatus">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>状态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogParameter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btDialogBootloader">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>更新程序</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="tbMessage">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>命令:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leCmd"/>
         </item>
         <item>
          <widget class="QPushButton" name="btExecute">
           <property name="text">
            <string>执行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btClearMessage">
           <property name="text">
            <string>清屏</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btDialogDataSource</tabstop>
  <tabstop>btDialogScene</tabstop>
  <tabstop>btDialogCalibration</tabstop>
  <tabstop>btDialogControl</tabstop>
  <tabstop>btDialogStatus</tabstop>
  <tabstop>btDialogParameter</tabstop>
  <tabstop>btDialogBootloader</tabstop>
  <tabstop>leCmd</tabstop>
  <tabstop>btExecute</tabstop>
  <tabstop>btClearMessage</tabstop>
  <tabstop>tbMessage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
